docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -511,6 +517,17 @@ The MongoDB POJO codec doesn't support `ZonedDateTime` and `OffsetDateTime` so y
 
 MongoDB with Panache also supports extended MongoDB queries by providing a `Document` query, this is supported by the find/list/stream/count/delete methods.
 
+MongoDB with Panache offers operations to update multiple documents based on an update document and a query :
+`Person.update("foo = ?1 and bar = ?2", fooName, barName).where("name = ?1", name)`.
```

Comment:
If it works with `,` instead of `and` as the separator I'd prefer we document that and leave the `and` syntax undocumented, because the `and` makes it look like a query rather than a set of update commands.

Reply:
Yes, it works, I tested it.
I just updated the documentation part.